README.md

```diff
@@ -1,25 +1,77 @@
 # 3scale-operator
 
+### Project Status: alpha
+
+The project is currently alpha which means that there are still new feautres and APIs planned that will be added in the future.
+Dueo to this breaking changes may still happen.
+
+Only use for short-lived testing clusters. Do not deploy in a project with a 3scale API Management solution.
+
+## Overview
+
+This project contains the 3scale operator software. 3scale operator is a
+software based on [Kubernetes operators](https://coreos.com/operators/) that
+provides:
+* A way to install a 3scale API Management solution, providing configurability
+  options at the time of installation
+* Ability to define 3scale API definitions and set
+  them into a 3scale API Management solution
+
 ## Prerequisites
 
 - [operator-sdk] version v0.2.1.
 - [dep][dep_tool] version v0.5.0+.
 - [git][git_tool]
 - Access to a Openshift v3.11.0+ cluster.
 
-## Quick Start
+## Getting started
 
-To download and prepare the environment for 3scale-operator:
+Download the 3scale-operator project:
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/3scale
-$ cd $GOPATH/src/github.com/3scale
-$ git clone https://github.com/3scale/3scale-operator
-$ cd 3scale-operator
-$ git checkout master
-$ make vendor
+mkdir -p $GOPATH/src/github.com/3scale
```

Comment:
Can't you just `go get -u && cd`?

Reply:
It seems that the structure of the project is not completely go-gettable (the project structure is generated by the operator-sdk new command):

```
msoriano@localhost:~$ go get -u github.com/3scale/3scale-operator
package github.com/3scale/3scale-operator: no Go files in /home/msoriano/exampleworkspace/src/github.com/3scale/3scale-operator
```

I've seen other projects doing it the way I wrote too (operator-sdk for example), maybe for the same reason.